addons/test_mail_full/tests/test_res_users.py

```diff
@@ -55,7 +53,7 @@ def test_deactivate_portal_users_blacklist(self):
         blacklists = self.env['phone.blacklist'].search([
```

Comment:
Search itself should be updated I guess, as it is not linked to test data anymore. Maybe we could add a 3rd test user, to check batch is effectively working (here if we break batch the test does not show it).